pkg/pillar/agentlog/agentlog.go

```diff
@@ -225,9 +226,22 @@ func writeOrLog(log *base.LogObject, w io.Writer, msg string) {
 	}
 }
 
-var listenDebugRunning atomic.Bool
+var (
+	listenDebugRunning  atomic.Bool
+	psiCollectorRunning atomic.Bool
+	psiCollectorCtx     context.Context
```

Comment:
```suggestion
	
```
I think this one doesn't need to be global.

Reply:
done